R/setup.R

```diff
@@ -78,6 +78,10 @@ make_profile <- function(system, user, repos, libpath) {
   if (!is.null(libpath)) {
     cat(".libPaths(", deparse(libpath), ")\n", sep = "", file = profile,
         append = TRUE)
+
+    libpath_env <- encodeString(paste(libpath, collapse = .Platform$path.sep), quote = '"')
```

Comment:
Why do we need to `encodeString` here?

Reply:
env vars themselves do not need quoting imo. You are  supposed to quote them in the  shell,  when  you use  them. (But  e.g.  not  in  R.)